.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+on:
+    - pull_request
+    - push
+
+name: build yii-debug-frontend
+
+jobs:
+    tests:
+        name: Node ${{ matrix.node-version }}-${{ matrix.os }}
+        runs-on: ${{ matrix.os }}
+
+        strategy:
+          matrix:
+            os:
+                - ubuntu-latest
+
+            node-version:
+                - 12.x
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Cache node modules
+              uses: actions/cache@v2
+              with:
+              path: ~/.npm
```

Comment:
He lacks spaced so he makes an error in in https://github.com/Julian-B90/yii-debug-frontend/actions/runs/471987817.

```yml
with:
  path:
```